test/e2e/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/gavv/httpexpect/v2 v2.2.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/gruntwork-io/terratest v0.32.8
-	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/ginkgo/v2 v2.1.4
```

Comment:
Can we use `replace` directive so that we can avoid a lot of changes?

Reply:
I don't think `replace` is the best practice for the gomod.
And if we use `replace`, it may lead to some confusion in the source code.

Reply:
I use an alias on the import, so it has already avoid as more changes as we can.

Reply:
I agree with @Fatpa 
I hope this project can be more clear, and there are not many changes, which have been done in this PR.